.env

```
@@ -1,2 +1,3 @@
-EXPO_BASE_URL = http://192.168.1.4:2000
+# EXPO_BASE_URL = http://192.168.1.4:2000
+EXPO_BASE_URL = http:/192.168.31.197:2000
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)